homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -95,6 +132,16 @@ async def async_setup_entry(
         SwissPublicTransportSensor(coordinator, description, unique_id)
         for description in SENSORS
     )
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_extra_state_attributes",
+        breaks_in_ha_version="2024.10.0",
+        is_fixable=False,
+        issue_domain=DOMAIN,
+        severity=IssueSeverity.WARNING,
+        translation_key="deprecated_extra_state_attributes",
+    )
```

Comment:
We don't have a good way to deprecate extra state attributes. So the general guideline is to just mention it in the change log and that's it

Reply:
Ahh, ok. I will remove and just out it into the breaking change part of the PR